### PR TITLE
Issue: 724 Make index.html <head> content theme specific

### PR DESCRIPTION
**Pull request by qiqqq**
I added vue-meta to project and removed current meta behavior (lib/meta.js, meta vuex store, meta tests and docs about meta vuex store).
Now each theme has resource/head.js file with theme-specific meta data.
Additionaly, each core component has default meta title and description rules.
The priority looks like this:

1) The resource/head.js file in theme has default meta data for theme
2) Title and description are overvriten by rules in core components
3) Title and description from core components are overwriten by meta data specified in rutes
4) Each data can be overwriten by data in theme component

### CODE REVIEW

**Comment by filrak** [comment]
Its just awesome. Great commit and i love the solution! Thanks!

File: src/index.template.html [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-    <meta charset="utf-8">
-    <title>{{ state.meta.title }}</title>
-    <meta name="description" content="{{ state.meta.description }}">
-    <meta name="mobile-web-app-capable" content="yes">
-    <meta name="viewport" content="width=device-width, initial-scale=1, minimal-ui">
-    <meta name="theme-color" content="#f60">
-    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+  	{{{ meta.inject().title.text() }}}
+  	{{{ meta.inject().meta.text() }}}
     <link rel="apple-touch-startup-image" href="/assets/launcher-icon-2x.png">
```
Comment (filrak) [inline src/index.template.html]: Can you also move this ones to `head.js` files?

**Comment by qiqqq** [comment]
I moved <links> to head.js (I left only links generated by webpack) and added some fixes to description tag

**Comment by filrak** [comment]
Great, thanks!

**Pull request merged**